netty/src/main/java/io/grpc/netty/NettyClientHandler.java

```diff
@@ -348,7 +350,11 @@ private void onRstStreamRead(int streamId, long errorCode) {
     if (stream != null) {
       Status status = GrpcUtil.Http2Error.statusForCode((int) errorCode)
           .augmentDescription("Received Rst Stream");
-      stream.transportReportStatus(status, false /*stop delivery*/, new Metadata());
+      stream.transportReportStatus(
+          status,
+          errorCode == Http2Error.REFUSED_STREAM.code() ? REFUSED : PROCESSED,
```

Comment:
nit: I'd prefer if the static import was avoided. This would be more clear knowing the type. Ditto elsewhere.

Reply:
Done.